### DISCUSSION TITLE
multi-level Multiapp variable transfer

### DISCUSSION THREAD

**Discussion by karthichockalingam** [idaholab/moose#24849] [Q&A General] [answered]
Hi,

Let's say you have hierarchical levels of Multiapps - can you transfer a variable from the bottom-most (leaf-level sub-app) directly to the master without having to transfer between the intermediate-level sub-apps?

Best,
Karthik.

**Comment by GiudGiud** [comment, marked as answer]
Hello

No we can do parent to grand-child-app transfers directly.

You could consider collapsing the hierarchy and have a parent/main app and make the grand-child app a child app of the parent/main app

Guillaume
